kasa/cli.py

```diff
@@ -347,7 +347,7 @@ async def usage(dev: SmartDevice, year, month, erase):
     if year:
         click.echo(f"== For year {year.year} ==")
         click.echo("Month, usage (minutes)")
-        usage_data = await usage.get_monthstat(year.year)
+        usage_data = await usage.get_monthstat(year=year.year)
```

Comment:
@jules43 I had to do change this to make the cli work, looks like has been broken for a while..

Reply:
Sadly I knew about that one but had only fixed it in a private version.